extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -56,14 +60,21 @@ require([
 			slotRegistry,
 			mercuryListener,
 			pageLevelParams.getPageLevelParams(),
+			adLogicZoneParams,
 			adContext,
 			btfBlocker,
 			'mercury',
-			trackingOptIn
+			trackingOptIn,
+			babDetection
 		);
 	});
+
 	win.loadCustomAd = adEngineBridge.loadCustomAd(customAdsLoader.loadCustomAd);
 
+	if (bidders) {
```

Comment:
I won't be undefined, never, am I wrong?

Reply:
Also, why do we need to call it outside `onLoad`?

Reply:
I forgot about `.isEnabled()` call. And also you right, we don't have to call it here.